cypress/e2e/WebInterface/Test Cases/QI-CORE Test Case/TestCaseExportImport/TestCaseImportValidations.cy.ts

```diff
@@ -563,13 +584,15 @@ describe('Test Case Import: New Test cases on measure validations: uniqueness te
 
         //import the tests cases from selected / dragged and dropped .zip file
         cy.get(TestCasesPage.importTestCaseBtnOnModal).click()
-        cy.get(TestCasesPage.testCaseListTable).should('contain.text', 'Case #StatusGroupTitleDescriptionLast SavedAction2N/ASBTestSeriesFb2Failing Test Caseb2' + secondTestCaseDescription + 'b2' + todaysDate + 'Select1N/ASBTestSeriesPb1Passing Test Caseb1' + testCaseDescription + 'b1' + todaysDate + 'Select')
-
+     
+        // checks for details of tc1 and tc2, but does not care about order
```

Comment:
I have made similar changes to other tests. This removes the check for the whole table (which needs the import to happen in the same order) and instead checks for the contents of test case 1 & test case 2 to be correct separately.